public/index.html

```diff
@@ -0,0 +1,5 @@
+<form action = '/submit'>
+  Name: <input type = 'text' name = 'a1'/>
```

Comment:
`name='a1'` is a bad name Just like variables, please come up with better naming.